artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -129,9 +129,9 @@
    protected static final int CRITICAL_CHECK_DEPAGE = 4;
 
    private static final Logger logger = Logger.getLogger(QueueImpl.class);
-   private static final AtomicIntegerFieldUpdater dispatchingUpdater = AtomicIntegerFieldUpdater.newUpdater(QueueImpl.class, "dispatching");
-   private static final AtomicLongFieldUpdater dispatchStartTimeUpdater = AtomicLongFieldUpdater.newUpdater(QueueImpl.class, "dispatchStartTime");
-   private static final AtomicLongFieldUpdater consumerRemovedTimestampUpdater = AtomicLongFieldUpdater.newUpdater(QueueImpl.class, "consumerRemovedTimestamp");
+   private static final AtomicIntegerFieldUpdater<QueueImpl> dispatchingUpdater = AtomicIntegerFieldUpdater.newUpdater(QueueImpl.class, "dispatching");
```

Comment:
What's the difference here? what's the advantage?

Reply:
This just solves a lot of (16) compiler warnings. When declared: "AtomicIntegerFieldUpdater is a raw type. References to generic type AtomicIntegerFieldUpdater<T> should be parameterized"
When referenced: "Type safety: The method get(Object) belongs to the raw type AtomicIntegerFieldUpdater. References to generic type AtomicIntegerFieldUpdater<T> should be parameterized"